tools/utils/http-helpers.js

```diff
@@ -169,6 +169,9 @@ _.extend(exports, {
 
     // This should never, ever be false, or else why are you using SSL?
     options.forceSSL = true;
+    if (process.env.CAFILE) {
+      options.ca = files.readFileSync(process.env.CAFILE);
```

Comment:
also, this should be `readFile` not `readFileSync` - it uses fibers to have a synchronous API while actually being non-blocking.